spec/models/configuration_script_base_spec.rb

```diff
@@ -0,0 +1,13 @@
+describe ConfigurationScriptBase do
+  it "has many authentications and vice versa" do
```

Comment:
haha this `btw` is really what I want to talk about 😄 

But trusting other developers' tests to cover your code seems shaky to me.  If this how Rails developers agree on, then we should remove a bunch existing specs e.g. [here](https://github.com/ManageIQ/manageiq/blob/c07352f9f07c9dc81a232a75104938d18522a501/spec/models/manageiq/providers/ansible_tower/configuration_manager/configuration_script_spec.rb#L10) which I think was from @bdunne 🙈 .  And I think there are more if we do a quick search.

Maybe this is another [mIq talk](talk.manageiq.org) topic 😄  ?